.licenserc.yaml

```diff
@@ -35,4 +35,29 @@ header:
 
 dependency:
   files:
-    - go.mod
\ No newline at end of file
+    - go.mod
+  licenses:
```

Comment:
Would these trigger dependencies check in the GHA process?

Reply:
No, GHA only checks the file header.